src/remote/impl/rms.external.controller.service.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpService } from '@nestjs/common';
+import { Logger } from 'protocol-common/logger';
+import { ProtocolHttpService } from 'protocol-common/protocol.http.service';
+import { IExternalControllerService } from '../external.controller.service.interface';
+
+export class RmsExternalControllerService implements IExternalControllerService {
```

Comment:
What is the `Rms` for?

Reply:
renamed to Rnp controller since its specific to Rnp